packages/ChangesReloaded-Core.package/CRBrowser.class/instance/createWindowSpecWith..st

```diff
@@ -0,0 +1,11 @@
+building
+createWindowSpecWith: aBuilder
+
+	| windowSpec |
+	windowSpec := aBuilder pluggableWindowSpec new
+		model: self;
```

Comment:
Why do you assign this to a temporary variable, just to return it immediately after assigning to it? It's several lines of boilerplate code for the benefit of what?
If this method would do more things I'd agree this would be sensible to let users know up-front that we are about to create a window spec, but the method is already named `createWindowSpecWith: aBuilder`, so if we would return something right away, readers would also know a window spec is being built.

Reply:
fair enough. we followed the legacy code first, then changed stuff which made the temp var useless and didn't think about that again